extension/src/triggers.rs

```diff
@@ -0,0 +1,56 @@
+use pgx::*;
+
+#[pg_extern]
+fn timescale_analytics_probe() {
+
+}
+
+extension_sql!(r#"
+CREATE OR REPLACE FUNCTION disallow_experimental_dependencies()
```

Comment:
Can/should we add a few tests around this? To make sure it's catching dependencies?

Reply:
It does, it broke a bunch of our tests 😋

Reply:
ha. well, I guess that works, but probably won't work in the future if we ever modify this?

Reply:
also added a must-fail test